apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -65,6 +66,8 @@ export interface AichatState {
   chatMessagePending?: ChatMessage;
   // Denotes whether we are waiting for a chat completion response from the backend
   isWaitingForChatResponse: boolean;
+  // Student events viewd by a teacher user in chat workspace
+  allStudentAichatEvents: AichatEvent[];
```

Comment:
nit: maybe `studentChatHistory: AichatEvent[]` might be clearer? or even `viewAsUserChatHistory` if we wanted to get even more generic

Reply:
I like `studentChatHistory`! Will update.